.github/scripts/install_start_yugabyte.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+
+set -exo pipefail
+export ARCHITECTURE=`uname -m`
+export MOS=`uname|tr '[:upper:]' '[:lower:]'`
+
+if [[ $ARCHITECTURE != "x86_64" && $ARCHITECTURE != "aarch64" ]]; then
+  echo "Arch $ARCHITECTURE not yet supported"
+  exit 1
+fi
+
+if [[ $ARCHITECTURE == "aarch64" ]]; then
+  export MOS="el8"
+fi
+
+show_help() {
+cat <<-EOT
+Usage: ${0##*/} <YB_VERSION_BUILD>
+EOT
+}
+
+YB_VERSION_BUILD=$1
+
+if [[ "${YB_VERSION_BUILD}x" == "x" ]]; then
+  show_help
+  exit 1
+fi
+
+YUGABYTE_SRC=${2:-/home/yugabyte}
+
+YB_VERSION=`echo $YB_VERSION_BUILD| awk -F'-' '{print $1}'`
+URL="https://downloads.yugabyte.com/releases/$YB_VERSION/yugabyte-$YB_VERSION_BUILD-$MOS-$ARCHITECTURE.tar.gz"
+
+yum -y -q install java-11-openjdk-devel
```

Comment:
well, I guess this won't get to far on ubuntu anyway so maybe the extra check above is less important...
Can we add 'centos' or 'rhel' or something to the script name?  Or maybe check that we are on the OS type we require and fail with a meaningful message if not?

Reply:
Updated script to support java installation on both rhel based and debian.